db/seeds.rb

```diff
@@ -101,4 +109,42 @@
   remote_picture = URI.open(article.picture)
   new_article.picture.attach(io: remote_picture, filename: article.picture_alt)
 end
-puts 'Articles have been added to each team from the AFC West subcategory!'
\ No newline at end of file
+puts 'Articles have been added to each team from the AFC West subcategory!'
+
+comments = [
+  OpenStruct.new(
+{content: "Great game!",
+    user: User.all[0],
+    article: Article.all[0]}),
+  OpenStruct.new(
+  {content: "What a waste of time!!!",
+   user: User.all[1],
+   article: Article.all[1]}),
+  OpenStruct.new(
+  {content: "Good game, boys!",
+   user: User.all[0],
+   article: Article.all[1]}),
+  OpenStruct.new(
+  {content: "REFERREEEE!!",
+   user: User.all[2],
+   article: Article.all[2]}),
+  OpenStruct.new(
+  {content: "A joy to watch the game in November. Ahhh ^_^",
+   user: User.all[1],
+   article: Article.all[2]})
+]
+
+comments.each do |comment|
+  Comment.create!(content: comment.content, user: comment.user, article: comment.article)
```

Comment:
no need to wrap hash into OpenStruct and then retrieve attributes, instead you could just pass hash directly into `create`